python/DEVELOPER.md

```diff
@@ -25,26 +25,39 @@ Software Dependencies
 
 ```bash
 sudo apt update -y
-sudo apt install -y python3 python3-venv git gcc pkg-config protobuf-compiler openssl libssl-dev
+sudo apt install -y python3 python3-venv git gcc pkg-config openssl libssl-dev unzip
+# Install rust
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 source "$HOME/.cargo/env"
+rustc --version
+# Install protobuf compiler
+curl -LO $PB_REL/download/v3.20.3/protoc-3.20.3-linux-x86_64.zip
+unzip protoc-3.20.3-linux-x86_64.zip -d $HOME/.local
+export PATH="$PATH:$HOME/.local/bin"
+protoc --version
 ```
 
 **Dependencies installation for CentOS**
 
 ```bash
 sudo yum update -y
-sudo yum install -y python3 git gcc pkgconfig protobuf-compiler openssl openssl-devel
+sudo yum install -y python3 git gcc pkgconfig openssl openssl-devel unzip
 pip3 install virtualenv
+# Install rust
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 source "$HOME/.cargo/env"
+# Install protobuf compiler
+curl -LO $PB_REL/download/v3.20.3/protoc-3.20.3-linux-x86_64.zip
```

Comment:
do you want to define what $PB_REL is supposed to be?

Reply:
good catch. thanks 👍